pkg/apis/externaldns/types.go

```diff
@@ -105,7 +105,7 @@ type Config struct {
 	BluecatDNSDeployType              string
 	BluecatSkipTLSVerify              bool
 	CloudflareProxied                 bool
-	CloudflareZonesPerPage            int
+	CloudflareDNSRecordsPerPage       int
```

Comment:
This is a breaking change. I think you should make it backwards compatible.

Reply:
@szuecs happy to leave that flag there - this was buried in the summary above:

> An old, unused config option around pagination of zones, CloudflareZonesPerPage, is also removed - I believe it was supposed to be removed on https://github.com/kubernetes-sigs/external-dns/pull/2298.

Note that this is a flag is a no-op.
Are you suggesting that it was left intentionally, so that folks using it could continue doing so without getting an error?

If that is the case, I'm happy to leave it around, or remove it in a separate PR. Let me know which one you prefer.

Reply:
If it was unused before then drop it as you did :+1: